bbc/simorgh
# Lite url transform refactor

Overall changes
======
- Refactors the Lite URL transform to use the `URL` class
- This allows more accurate adding of the `.lite` extension in URLs that may have query params and hashes in the URL

Testing
======
1. Visit http://localhost:7080/gahuza.lite?renderer_env=live
2. Scroll to the very bottom and hover over the `Kanda hano` button in the message banner
3. Confirm the `.lite` extension is in the correct position and not at the very end of the URL as is found on https://www.bbc.com/gahuza.lite - note the page will 404 when clicked in the new version as /ws/languages.lite routed yet

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
